docs/source/terms_and_model.rst

```diff
@@ -379,9 +375,49 @@ A contiguous region specified by named features.
    {
      'end': 'q22.3',
      'start': 'q22.2',
-     'type': 'NamedInterval'
+     'type': 'CytobandInterval'
    }
 
+.. _TextInterval:
+
+TextInterval
+$$$$$$$$$$$$$$
+
+**Computational definition**
+
+A contiguous region specified by named features.
```

Comment:
This definition seems to make TextInterval a superclass of CytobandInterval, so additional clarity between these classes might be helpful (e.g., explicitly noting TextInterval should not be used to express Cytobands).  Do we have use cases for TextInterval, or is it introduced simply for parallelism?

Reply:
I am struggling a bit with the breadth of TextInterval. It was a bit of a late addition to our thinking that, IMO, we didn't reach consensus on before including here.

Is there an immediate use case in 1.1? If not, I would prefer that we yank it and reconsider in the context of other classes.